src/js/config/config.js

```diff
@@ -20,13 +20,4 @@ var config = {
     `${packageJSON.version}-SNAPSHOT`
 };
 
-if (process.env.GULP_ENV === "development") {
-  try {
-    var configDev = require("./config.dev");
-    config = Object.assign(config, configDev);
-  } catch (e) {
-    console.info("You could copy config.template.js to config.dev.js " +
-      "to enable a development configuration.");
-  }
-}
-module.exports = config;
```

Comment:
@orlandohohmeier removing this block unfortunately makes the non-overridden config options (for example, `rootUrl`) in the dev config file `undefined`. I can't see the logo anymore, that's how I found out!